Bottom: dcc79fc5f3a9aac43ef60099515a4551b77c0a71
Top:    f33fe881847aa508a15622dc561b81206586d595
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-06 16:26:01 +0000

Refresh of major-re-write-to-drawing-rout

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 2e0502c..d6d1e83 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -1352,8 +1352,8 @@ DrawLayerGroup (int group, const BoxType * screen)
     layernum = layers[i];
     Layer = PCB->Data->Layer + layers[i];
 
-    if (strcmp (Layer->Name, "outline") == 0 ||
-        strcmp (Layer->Name, "route") == 0)
+    if (strcasecmp (Layer->Name, "outline") == 0 ||
+        strcasecmp (Layer->Name, "route") == 0)
       rv = 0;
 
     if (layernum < max_layer /*&& Layer->On*/) {
